spec/models/version_spec.rb

```diff
@@ -150,7 +150,7 @@
         expect(subject.production?).to be_falsey
         expect(subject.preview?).to be_truthy
         expect(subject.latest_preview?).to be_truthy
-        expect(subject.publishable?).to be_truthy
+        expect(subject.publishable?).to be_falsey
```

Comment:
Can we fix up these `version` specs?  
This is testing if a `Version.current_preview.publishable?` should be truthy.  It _should_ be truthy, right?
I think you need to fix this in the `versions` factory by setting a value for `completed_at`.

Along those lines, maybe you need another scope to make return records where `completed_at` is not `nil`  to be applied where appropriate.  First place I see is `Version.current_preview`.  Perhaps the definition of that method should be `Version.preview.completed.newest.first` (assuming you name the scope `completed`).  Also use it anywhere else you don't want a non-completed version to be returned.

Reply:
We do want `current_preview` to return the most recent preview version regardless of whether or not is has been completed, so having `Version.current_preview.publishable?` be false is correct here.

I can add an additional test to cover the case where the current_preview version is completed.  Does that work?

Reply:
I see.  Yes please to covering when `completed_at` is not `nil`.